client/app/cdn/dedicated/domain/rule/order/cdn-dedicated-domain-rule-order.html

```diff
@@ -1,133 +1,129 @@
-<div data-ng-controller="CacherulesAddCtrl">
-
-    <div data-wizard
-         data-wizard-on-cancel="resetAction"
-         data-wizard-on-finish="displayBC"
-         data-wizard-title=":: 'cdn_configuration_cacherules_upgrade_title' | translate"
-         data-wizard-bread-crumb
-         data-wizard-confirm-button-text=":: 'wizard_pay' | translate">
-
-        <div data-wizard-step
-             data-wizard-step-on-load="loadCacherulesPrice"
-             data-wizard-step-valid="choices.count">
-
-            <div class="text-center"
-                 data-ng-if="cacheRulesLoading">
-                <oui-spinner></oui-spinner>
-            </div>
+<div data-wizard
```

Comment:
Only previous inline `data-ng-controller` has been changed in this file (+ adding $uibModalInstance.dismiss)